assets/src/application/ui/display/EmptyState.tsx

```diff
@@ -1,32 +1,28 @@
-/**
- * External dependencies
- */
 import * as React from 'react';
 import { Empty } from 'antd';
 
 interface EmptyStateProps {
-	children: React.ReactChild;
+	children?: React.ReactNode;
+	className?: string;
 	description: string;
-	title: string;
+	title?: string;
 }
 
-const EmptyState = (props: EmptyStateProps) => {
-	const { children, description, title } = props;
-
+const EmptyState = ({ className, children, description, title }: EmptyStateProps) => {
 	return (
-		<>
+		<div className={className}>
 			<Empty
 				description={
 					<>
-						{title}
+						{title && title}
```

Comment:
`{title && title}` is equivalent to just `{title}` if it's always going to be a either a valid React Node or (`null` | `undefined` | `false`). If you think it can be otherwise, then even `{title && title}` won't help because if `title` is a truthy value e.g. a function, it will cause an error.

Reply:
> {title && title} is equivalent to just {title}

`title` type is optional string, it can be `undefined` or a string